inc/class-cachify.php

```diff
@@ -132,6 +132,30 @@ public function __construct() {
 			2
 		);
 
+		/* Add Cron for clearing the HDD Cache */
+		if ( self::METHOD_HDD == self::$options['use_apc'] ) {
+			add_filter(
+				'cron_schedules',
+				array(
+					__CLASS__,
+					'add_cron_cache_expiration',
+				)
+			);
+
+			$timestamp = wp_next_scheduled( 'hdd_cache_cron' );
+			if (false == $timestamp) {
```

Comment:
Same here, === instead of ==

Reply:
Is included in "Make requested changes to checks"